backend/routes/getters.js

```diff
@@ -39,4 +66,79 @@ router.get("/companyById/:id", async (req, res) => {
   );
 });
 
+// get company documents ID!
+router.get("/documents/:id", async (req, res) => {
+  const id = parseInt(req.params.id);
+  res.json(
+    await prisma.document.findMany({
+      where: {
+        companyId: id,
+      },
+      orderBy: {
+        filed_date: "desc",
+      },
+    })
+  );
+});
+
+// get company news data!
+router.get("/news/:id", async (req, res) => {
+  const id = parseInt(req.params.id);
+
+  const company = await prisma.company.findUnique({
+    where: {
+      id,
+    },
+  });
+
+  if (company == null) {
+    res.status(404).json({
+      message: "company is not in database / is not pubically traded",
+    });
+  }
+  const companyName = company.name;
+
+  let currentArticles = await prisma.article.findMany({
+    where: { companyId: id },
+    orderBy: { created_at: "desc" },
+  });
+  if (currentArticles.length !== 0) {
+    if (currentArticles[0].created_at - Date.now() < 18000000) {
```

Comment:
readability nit: sometimes it makes sense to write these large numbers as their constituent parts for easier reading. This appears to mean 18M milliseconds which is 5 * 3600 * 1000 or 5 * 60 * 60 * 1000, and I'd say the meaning of both is more legible to the average reader.

You've already got a comment explaining this so it's not a huge deal :)